pystarport/cluster.py

```diff
@@ -214,6 +214,7 @@ def create_node(
         # init home directory
         self.init(i)
         home = self.home(i)
+        mode = self.config.get("broadcast-mode", "sync")
```

Comment:
I guess we can add a `client-config` field.

Reply:
I think there's one already: `client_config`.

Reply:
seems we need allow custom when [create_node](https://github.com/crypto-org-chain/chain-main/blob/master/integration_tests/test_staking.py#L135)